src/components/task/create-new/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import Button from "components/common/button";
+import TextArea from "components/common/text-area";
+import Image from "components/common/image";
+import { addTaskToTodo } from "store/actions/todo";
+import { generateTaskObject, sanitizer } from "utils/helper";
+import { ERROR_MESSAGE_TASK_TITLE } from "utils/const/errorMessages";
+import { ATTRIBUTE_TITLE, BUTTON_TYPE_BUTTON } from "utils/const/formElements";
+import { ALT_TAG_ICON_DELETE, ICON_DELETE } from "utils/const/images";
+import styles from "components/task/create-new/index.module.scss";
+
+function CreateTask({ onSuccessfullTaskEntry }) {
+  const dispatch = useDispatch();
+  const [errorMessage, setErrorMessage] = useState("");
+
+  function handleAddTaskSubmit(event) {
+    event.preventDefault();
+    const title = event.target.title.value;
+    const sanitizedTitle = sanitizer(title);
+    if (sanitizedTitle === "") {
+      setErrorMessage(ERROR_MESSAGE_TASK_TITLE);
+      return;
+    }
+    dispatch(addTaskToTodo(generateTaskObject(sanitizedTitle)));
+    onSuccessfullTaskEntry();
```

Comment:
form data should be passed back to the parent to handle application state update.